tests/test_data_file.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import mock
+import struct
+
+import sys
+
+# Used for file access emulation.
+import StringIO
+
+import common
+import eblob_kit
+
+
+TEST_OFFSETS_RANGE = 7
+
+
+if (sys.version_info < (3, 0)):
```

Comment:
https://docs.python.org/3.3/library/unittest.mock.html#mock-open

Reply:
It is needed sequential file reads to be emulated in most tests here, unfortunately I haven't found so far any simple way to implement such behavior with `mock_open`.